### PR TITLE
Add extra debug so we can tell apart postdelete hooks

### PR DESCRIPTION
**Pull request by mheon**
Trivial patch to add some debug so we can tell which postdelete hooks are throwing errors.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: libpod/container_internal.go [hunk 1]
```diff
@@ -913,7 +913,7 @@ func (c *Container) postDeleteHooks(ctx context.Context) (err error) {
 				return err
 			}
 			for i, hook := range extensionHooks {
-				logrus.Debugf("container %s: invoke poststop hook %d", c.ID(), i)
+				logrus.Debugf("container %s: invoke poststop hook %d, path %s", c.ID(), i, hook.Path)
```
Comment (wking) [inline libpod/container_internal.go]: If this is for grepping logs, do we want to add it to the debug/warning lines below as well?

Comment (mheon) [reply libpod/container_internal.go]: I think once is sufficient - the failure, if any, is logged on the next line

Comment (wking) [reply libpod/container_internal.go]: > I think once is sufficient - the failure, if any, is logged on the next line

That works if folks use `-A1` or similar in their log grep, but doesn't help if your log level is lower than "debug".  Maybe add it to the following `Warnf` as well?

Also, if you want to extract the hook from existing debug logs, you can get the matched hooks from [here][1].

[1]: https://github.com/projectatomic/libpod/blob/v0.6.3/pkg/hooks/hooks.go#L116

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**